infra-as-code/bicep/modules/vnetExisting.bicep

```diff
@@ -0,0 +1,15 @@
+@description('This is the name of the vnet')
+param vnetName string
+
+@description('This is the name of the subnet you want the id for')
+param subnetName string
+
+resource vnet 'Microsoft.Network/virtualNetworks@2019-11-01' existing = {
+  name: vnetName
+
+  resource subnet 'subnets' existing = {
+    name: subnetName
+  }
+}

```

Comment:
This pattern of doing `existing` in modules and that's all the module does is still so weird for me.  I'm torn :)  [No action item here, just talking outloud]